middlewares/validators/user.js

```diff
@@ -52,7 +52,7 @@ const updateUser = async (req, res, next) => {
     site: joi.string().optional(),
     isMember: joi.boolean().optional(),
     tokens: joi.object().optional(),
-    website: joi.string().optional()
+    website: joi.string().optional().allow('')
```

Comment:
Why is this change being made?
The website name shouldn't be an empty string, right?

Reply:
Yes. Currently the signup form has `website` as optional and when the user leaves it blank, the backend sends 400 response saying `website cannot be empty` 

Source: https://github.com/sideway/joi/blob/v10.6.0/API.md#string---inherits-from-any
![image](https://user-images.githubusercontent.com/56217868/106610729-1ad6a200-658d-11eb-9ae0-ecefb0dd42bd.png)

Reply:
Yeah, but this doesn't mean that we should explicitly allow the empty strings. This is an issue from the UI. The UI shouldn't be sending empty strings for website value.